### PR TITLE
feat: added network and volume integration (fix: #33)

### PR DESCRIPTION
**Pull request by mxschmitt**
Addresses issue #33

### CODE REVIEW

**Comment by LKaemmerling** [comment]
Just as an Idea, why not let the user specify the name of the network/volume too?  You could use https://github.com/hetznercloud/hcloud-go/blob/master/hcloud/network.go#L99   and https://github.com/hetznercloud/hcloud-go/blob/master/hcloud/volume.go#L79  for this :) Otherwise looks good from my perspective :)

**Comment by mxschmitt** [comment]
> Just as an Idea, why not let the user specify the name of the network/volume too? You could use [hetznercloud/hcloud-go:hcloud/network.go@`master`#L99](https://github.com/hetznercloud/hcloud-go/blob/master/hcloud/network.go#L99) [](/hetznercloud/hcloud-go/blob/HEAD@{2019-07-10T12:01:53Z}/hcloud/network.go#L99) and [hetznercloud/hcloud-go:hcloud/volume.go@`master`#L79](https://github.com/hetznercloud/hcloud-go/blob/master/hcloud/volume.go#L79) [](/hetznercloud/hcloud-go/blob/HEAD@{2019-07-10T12:01:53Z}/hcloud/volume.go#L79) for this :) Otherwise looks good from my perspective :)

would you add e.g. `hetzner-volume-ids` and `hetzner-volume-names`? So that there are two parameters for it? Not sure, but in general good catch!

**Comment by LKaemmerling** [comment]
I guess I would leave it only `hetzner-volumes` (e.g. `hetzner-networks`)

**Comment by mxschmitt** [comment]
> I guess I would leave it only `hetzner-volumes` (e.g. `hetzner-networks`)

But then is the handling to determine if it's a name or an ID not so ideal. Just try to parse it as a number and if it fails try to get by name is quite messy in my mind, hmm.

**Comment by LKaemmerling** [comment]
You don't need to parse it :) We do this in the `hcloud-go` already for you. Just allow the user to input strings.

**Comment by mxschmitt** [comment]
> You don't need to parse it :) We do this in the `hcloud-go` already for you. Just allow the user to input strings.

That's very neat, thanks for the tip!

File: driver.go [hunk 1]
```diff
@@ -37,6 +37,8 @@ type Driver struct {
 	danglingKey    bool
 	ServerID       int
 	userData       string
+	volumeIDs      []string
```
Comment (LKaemmerling) [inline driver.go]: i would rename it to only `volumes`
```suggestion
      volumes        []string

**Comment by LKaemmerling** [comment]
LGTM!

**Pull request merged**